tests/ltp/patches/fix_sigrelse_sigrelse01.patch

```diff
@@ -0,0 +1,673 @@
+This test case creates a child process using "FORK_OR_VFORK".
```

Comment:
Please change patch file name to sigrelse01.patch 
You can run command
git mv fix_sigrelse_sigrelse01.patch sigrelse01.patch

Reply:
@hukoyu, I renamed the file to sigrelse01.patch.